### PR TITLE
Correct typo/grammar for Trackr removal

### PR DESCRIPTION
**Pull request by FuzzyMistborn**
## Proposed change

Correct typo in release notes for Trackr integration.  Hope I did this against the correct branch.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_posts/2021-10-06-release-202110.markdown [hunk 1]
```diff
@@ -496,8 +496,8 @@ installed, make sure it's part of your manifest.
 
 The following integrations are no longer available as of this release:
 
-- **Trackr** has been removed. The API is no longer available.
-  has therefore been removed. ([@FuzzyMistborn] - [#55917])
+- **Trackr** has been removed. The API is no longer available and
```
Comment (frenck) [inline source/_posts/2021-10-06-release-202110.markdown:499]: This is now duplicate, as the title of the list already implies an integration.

Comment (FuzzyMistborn) [reply source/_posts/2021-10-06-release-202110.markdown:499]: Ok, but there's still a typo.

Comment (frenck) [reply source/_posts/2021-10-06-release-202110.markdown:499]: Oh right!

File: source/_posts/2021-10-06-release-202110.markdown [hunk 2]
```diff
@@ -496,8 +496,8 @@ installed, make sure it's part of your manifest.
...
+  the integration has therefore been removed. ([@FuzzyMistborn] - [#55917])
```
Comment (frenck) [inline source/_posts/2021-10-06-release-202110.markdown]:
```suggestion
- **Trackr** has been removed. The API is no longer available and
  has therefore been removed. ([@FuzzyMistborn] - [#55917])
```

Comment (FuzzyMistborn) [reply source/_posts/2021-10-06-release-202110.markdown]: Thanks!

Comment (frenck) [reply source/_posts/2021-10-06-release-202110.markdown]: No thank you!

**Review by frenck — APPROVED** [review]
Thanks, @FuzzyMistborn 👍 

Happy Hacktoberfest 🎉 

../Frenck

**Pull request merged**